tracer/test/snapshots/Security.AspNetCore2.__test=blocking_enableSecurity=True_expectedStatusCode=403_url=_.verified.txt

```diff
@@ -17,8 +17,7 @@
       http.request.headers.host: localhost:00000,
       http.request.headers.user-agent: Mistake Not... Hello/V,
       http.request.headers.x-forwarded: 86.242.244.246,
-      http.response.headers.content-length: 2xxx,
-      http.response.headers.content-type: text/html,
+      http.response.headers.content-type: application/json,
```

Comment:
Is this change expected? 🤔

Reply:
yes it's a fix to the bug where by default we should respond with json